recipes/tokyocabinet/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "tokyocabinet" %}
+{% set version = "1.4.48" %}
+{% set sha256 = "a003f47c39a91e22d76bc4fe68b9b3de0f38851b160bbb1ca07a4f6441de1f90" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://fallabs.com/tokyocabinet/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - toolchain
+    - zlib 1.2.*
+    - bzip2 1.0.*
+  run:
+    - zlib 1.2.*
+    - bzip2 1.0.*
+
+test:
+  commands:
+    - test -e $PREFIX/include/tcadb.h  # [unix]
+    - test -e $PREFIX/lib/libtokyocabinet.so  # [linux]
+    - test -e $PREFIX/lib/libtokyocabinet.dylib  # [osx]
+    - test -e $PREFIX/lib/libtokyocabinet.a  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/tokyocabinet.pc  # [unix]
+    - tcbmgr version > /dev/null  # [unix]
+    - echo foo | tcucodec zlib > /dev/null  # [unix]
```

Comment:
I somehow remember reading that the tests should not write useless things to stdout, but maybe that was some other context. I can remove those...